### PR TITLE
Added my own scripts

### PR DESCRIPTION
**Pull request by kurobeats**
Just submitting my own zap scripts for possible addition

### CODE REVIEW

**Comment by kingthorin** [comment]
These look good to me. If I had to say something it would be that "Ignore cookies.zst " doesn't really ignore, it strips set-cookie headers. So nothing will be set while this is in-place but if something is already set then it still goes....

File: passive/SQL injection detection.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    ibmdb2 = /(CLI Driver.*DB2|DB2 SQL error|\bdb2_\w+\(|SQLSTATE.+SQLCODE)/g
+    informix = /(Exception.*Informix)/g
+    firebird = /(Dynamic SQL Error|Warning.*ibase_.*)/g
+    sqlite = /(SQLite\/JDBCDriver|SQLite.Exception|System.Data.SQLite.SQLiteException|Warning.*sqlite_.*|Warning.*SQLite3::|\[SQLITE_ERROR\])/g
+	sapdb = /(SQL error.*POS([0-9]+).*|Warning.*maxdb.*)/g
+	sybase = /(Warning.*sybase.*|Sybase message|Sybase.*Server message.*|SybSQLException|com\.sybase\.jdbc)/g
+	ingress = /(Warning.*ingres_|Ingres SQLSTATE|Ingres\W.*Driver)/g
+	frontbase = /(Exception (condition )?\d+. Transaction rollback.)/g
+	hsqldb = /(org\.hsqldb\.jdbc|Unexpected end of command in statement \[|Unexpected token.*in statement \[)/g
+
```
Comment (bowserjklol) [inline passive/SQL injection detection.js]: For your consideration: IMO, these regexs should be dropped into an array:

```JavaScript

dbmsErr = [mysql, postgresq, mysql, etc, etc]

```

Then you can just do one FOR loop to knock through all of these regexp exec calls.

```JavaScript

	for (var i=0; i < dbmsErr.length; i++)
	{
		while (comm = dbmsErr[i].exec(body)) 
		{
			foundSqlErr.push(comm[0]); 
		}

	}

```

Then alert using alertTitle[i] and alertDesc[i] and you chop out all of those Copy/Pasted and nearly identical IF tests.  Food for thought.

File: passive/SQL injection detection.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+		 "An Ingress error was discovered",
+		 "A Frontbase error was discovered",
+		 "A HSQLDB error was discovered",
+		""]
+    alertSolution = ["Ensure proper sanitisation is done on the server side, or don't. I don't care.",
+		    ""]
+    cweId = [0,1]
+    wascId = [0,1]
+
+    mysql = /(SQL syntax.*MySQL|Warning.*mysql_.*|MySqlException \(0x|valid MySQL result|check the manual that corresponds to your (MySQL|MariaDB) server version|MySqlClient\.|com\.mysql\.jdbc\.exceptions)/g
```
Comment (bowserjklol) [inline passive/SQL injection detection.js]: Totally minor but would recommend going case insensitive on each of these (/i)

**Comment by kingthorin** [comment]
@kurobeats did you ever get anything further on this?

**Comment by kurobeats** [comment]
@kingthorin I merged your changes, I don't recall any more movement...

**Comment by kingthorin** [comment]
@kurobeats it wasn't me that suggested changes. However I do see that you accepted some from the same user. Are you planning to revise this PR?

**Comment by thc202** [comment]
@kurobeats any news on this one?

**Comment by kingthorin** [comment]
@kurobeats any updates coming?

**Comment by kurobeats** [comment]
Nope

**Comment by kingthorin** [comment]
@psiinon @thc202 are you concerned about getting these mods? Do you want to merge or close?

**Comment by thc202** [comment]
FTR, I'm fine with the merge as well.

**Pull request merged**